### PR TITLE
PLANET-6091 Add new XXL breakpoint

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6091](https://jira.greenpeace.org/browse/PLANET-6091)

This commit also creates the corresponding mixin for easier CSS rules.
We also need to update: 
- The donation pages: [PR](https://github.com/greenpeace/planet4-en-petitions/pull/8)
- Some blocks (CarouselHeader, Columns, Gallery) content width: [PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/716)

### Testing

You can test the changes on this [test instance](https://www-dev.greenpeace.org/test-jupiter/) or on local.

### CODE REVIEW

**Comment by Inwerpsel** [comment]
I added "DO NOT MERGE" label to avoid issues with the [new post designs](https://github.com/greenpeace/planet4-master-theme/pull/1454). We should only merge this once that branch is merged and deployed.

**Review by lithrel — APPROVED** [review]
:+1:

File: assets/src/scss/pages/_page.scss
Comment (mleray) [inline assets/src/scss/pages/_page.scss:10]: Leaving the padding on bigger screen sizes was causing the content to be smaller than the expected width

File: assets/src/scss/pages/_page.scss
Comment (mleray) [inline assets/src/scss/pages/_page.scss:6]: I changed the value here to 16px instead of 15px to follow our new 8px grid

File: assets/src/scss/pages/post/_post.scss
Comment (mleray) [inline assets/src/scss/pages/post/_post.scss:129]: We should keep the Bootstrap values for padding

**Review by lithrel — APPROVED** [review]
:+1:

File: assets/src/scss/vendors/_bootstrap.scss
Comment (Inwerpsel) [inline assets/src/scss/vendors/_bootstrap.scss:7]: The ticket mentions "New breakpoint on extra extra large screen ≥1920px". If we want to match full HD screens, we should set this value a bit lower to account for small amounts of spaces the OS can subtract from the browser window (e.g. sidebar).

Bootstrap itself sets this value to 1400px by default, probably intending to match 1440px and up.

Comment (Inwerpsel) [reply assets/src/scss/vendors/_bootstrap.scss:7]: I remember we discussed this during the refinement meeting, but somehow we didn't capture it on the ticket :thinking:

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**
